puppet/modules/kubernetes/templates/kube-apiserver.service.erb

```diff
@@ -17,6 +17,12 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/apiserver \
 <% else -%>
   --allow-privileged=false \
 <% end -%>
+<% if @audit_enabled -%>
+  --audit-policy-file=<%= scope['kubernetes::apiserver::audit_policy_file'] %> \
+  --audit-log-path=<%= scope['kubernetes::apiserver::audit_log_path'] %> \
+  --audit-log-maxbackup=0 \
+  --audit-log-maxsize=100 \
```

Comment:
That means don't store rolled over files and roll over logs after they have 100 megs